app/controllers/stories/feeds_controller.rb

```diff
@@ -34,7 +34,10 @@ def assign_feed_stories
     end
 
     def signed_in_base_feed
-      if Settings::UserExperience.feed_strategy == "basic"
+      feed_variant = field_test(:feed_strategy, participant: current_user)
```

Comment:
Side note: we often wrap (external) dependencies in custom classes (`Flipper` -> `FeatureFlag`, `ENV`/related config stores -> `ApplicationConfig`, so maybe we should start doing something similar for `FieldTest`. I just whipped something up real quick to illustrate the idea:

```ruby
class ABTest
  extend FieldTest::Helpers

  def self.feed_strategy_for(user)
    field_test(:feed_strategy, participant: user)&.inquiry
  end
end

feed_variant = ABTest.feed_strategy_for(user)
#=> "original"
feed_variant.weighted_query_strategy?
#=> false
feed_variant.original?
#=> true
```

Obviously, this doesn't need to be part of this PR (it probably shouldn't), I was just thinking out loud basically.

Reply:
This perfectly aligns with where I wanted to go…and will make something much easier to do.

So…yeah, I'm gonna introduce this.

Reply:
I had been considering a `Articles::Feeds.strategy_for(user:)` implementation which would remove the case logic from the controller.  However, this `ABTestService` is something that's both useful to model as well a useful consideration.  It also doesn't supersede the idea of using a service type class to get the feed, be it based on a field test or site wide configuration or user configuration.